RadioItem/RadioItem.module.less

```diff
@@ -18,10 +14,28 @@
 			border-radius: @agate-radioitem-icon-border-radius;
 			color: transparent;
 			opacity: @agate-radioitem-icon-opacity;
+			margin: @agate-radioitem-icon-margin;
+			display: flex;
+			align-items: center;
+			justify-content: center;
```

Comment:
Why did you introduce this?
I think we could get the same result with `margin: 15px` here without display: flex.

Reply:
👍